site/content/en/docs/tasks/run_rayclusters.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Run A RayCluster"
+date: 2024-01-17
+weight: 6
+description: >
+  Run a RayCluster on Kueue.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [RayCluster](https://docs.ray.io/en/latest/cluster/getting-started.html).
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+Make sure you are using Kueue v0.6.0 version or newer and Kuberay 1.1.0 or newer.
+
+1. By default, the integration for `ray.io/raycluster` is not enabled.
```

Comment:
Is it really disabled by default. The `config/components/manager/controller_manager_config.yaml` has it enabled right?

Reply:
I think this is true for RayJob too, we should probably update both docs as a follow-up